packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
-{% endhint %}
+### Deploy a validator
 
-In order to allow the node to sync with the network, give it the address of existing nodes in the network to the Proxy. The Validator will sync through the Proxy:
+To actually register as a validator, we'll need to generate a validating signer key. On your Validator machine (which should not be accessible from the public internet), follow very similar steps:
 
 ```bash
-docker run -v $PWD/proxy:/root/.celo --entrypoint cp $CELO_IMAGE /celo/static-nodes.json /root/.celo/
+# On the validator machine
+mkdir -p celo-data-dir/validator
+cd celo-data-dir/validator
+docker run -v $PWD/validator:/root/.celo $CELO_IMAGE init /celo/genesis.json
+docker run -v $PWD/validator:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
+export CELO_VALIDATOR_SIGNER_ADDRESS=<YOUR-VALIDATOR-SIGNER-ADDRESS>
 ```
 
-#### Running the Proxy
-
-At this point we are ready to start up the Proxy:
+As the Valdiator account, we'll need to create a proof-of-possession of the Validator signer key with `celocli`. For that we can run the node quickly just to sign:
 
 ```bash
-docker run --name celo-proxy -d --restart always -p 8555:8545 -p 8556:8546 -p 30313:30303 -p 30313:30303/udp -p 30503:30503 -p 30503:30503/udp -v $PWD/proxy:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal,istanbul --etherbase=$CELO_PROXY_ADDRESS --proxy.proxy --proxy.proxiedvalidatoraddress $CELO_VALIDATOR_ADDRESS --proxy.internalendpoint :30503
+# On the validator machine
+docker run --name celo-validator -d --restart always -p 8545:8545 -v $PWD/accounts:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal,istanbul --unlock $CELO_VALIDATOR_SIGNER_ADDRESS
 ```
 
-#### Running the Validator
-
-Now that we have the Proxy up and running, we need to obtain its enode and IP address, so that we can connect the Validator to the proxy. You can do that running the following commands:
+Then create the proof-of-possession of the validator signer with:
 
 ```bash
-export PROXY_ENODE=$(docker exec celo-proxy geth --exec "admin.nodeInfo['enode'].split('//')[1].split('@')[0]" attach | tr -d '"')
-export PROXY_IP=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' celo-proxy)
+# On the validator machine
+# You have to export CELO_VALIDATOR_ADDRESS on this machine
+export $CELO_VALIDATOR_ADDRESS=<CELO_VALIDATOR_ADDRESS>
+celocli account:proof-of-possession --signer $CELO_VALIDATOR_SIGNER_ADDRESS --account $CELO_VALIDATOR_ADDRESS
 ```
 
-Now we can start up the Validator node. In the below command remember to replace the **VALIDATOR_ADDRESS_PASSWORD** for the password you used when you created the `CELO_VALIDATOR_ADDRESS`:
+For consensus, we are deriving a BLS key from the ECDSA key, we'll need to proof that we own the BLS key as well:
```

Comment:
Validators on the Celo network use BLS aggregated signatures to create blocks in addition to the Validator signer (ECDSA) key. While an independent BLS key can be specified, the simplest thing to do is to derive the BLS key from the Validator signer key. When we register our Validator, we'll need to prove possession of the BLS key as well, which can be done by running the following command: